yarn.lock

```diff
@@ -19469,14 +19629,14 @@ wrap-ansi@^6.2.0:
     string-width "^4.1.0"
     strip-ansi "^6.0.0"
 
-wrap-ansi@^7.0.0:
-  version "7.0.0"
-  resolved "https://registry.yarnpkg.com/wrap-ansi/-/wrap-ansi-7.0.0.tgz#67e145cff510a6a6984bdf1152911d69d2eb9e43"
-  integrity sha512-YVGIj2kamLSTxw6NsZjoBxfSwsn0ycdesmc4p+Q21c5zPuZ1pl+NfxVdxPtdHvmNVOQ6XSYG4AUtyt/Fi7D16Q==
+wrap-ansi@^8.1.0:
+  version "8.1.0"
+  resolved "https://registry.yarnpkg.com/wrap-ansi/-/wrap-ansi-8.1.0.tgz#56dc22368ee570face1b49819975d9b9a5ead214"
+  integrity sha512-si7QWI6zUMq56bESFvagtmzMdGOtoxfR+Sez11Mobfc7tm+VkUckk9bW2UeffTGVUbOksxmSw0AA2gs8g71NCQ==
   dependencies:
-    ansi-styles "^4.0.0"
-    string-width "^4.1.0"
-    strip-ansi "^6.0.0"
+    ansi-styles "^6.1.0"
+    string-width "^5.0.1"
```

Comment:
This seems to be when the faulty dependency breaking the builds gets pulled in

Reply:
what do we need to do here? pin string-width til 4.x? Upgrade all dependencies that pulls in `string-width`?

Reply:
Probably pinning it to 4.x. It seems like some dependency that ships as CJS is pulling it in as ESM in a way that is faulty/wrong. Team ecosystem have seen a lot of problems like this and we maintain a list over them for Renovatebot to skip making PRs: https://github.com/sanity-io/renovate-presets/blob/main/ecosystem/workaround-esm.json

Reply:
@RitaDias yes, this PR is too noisy